etc/dehydrated/hook.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
Yeah I think you're right on that. Chaining the calls with `&&` might be better.